test/fixture/install-global.js

```diff
@@ -0,0 +1 @@
+global.foo = 'bar'
```

Comment:
Weird. I love my semi-colons, too. I wonder why ESLint didn't bust my chops about this? :)